samples/ControlCatalog/Pages/PointersPage.cs

```diff
@@ -95,7 +95,7 @@ protected override void OnPointerCaptureLost(PointerCaptureLostEventArgs e)
 
         public override void Render(DrawingContext context)
         {
-            context.FillRectangle(Brushes.Transparent, new Rect(default, Bounds.Size));
+            context.DrawRectangle(Brushes.Transparent, null, new Rect(default, Bounds.Size));
```

Comment:
This change and the other changes in ControlCatalog can be reverted now, as we've kept the old API in place.